### PR TITLE
feat: see through lens

### LINKED ISSUE #1053: See the feed through the lens of someone else
Sometimes it's easy to get stuck in a bubble of only the people you're following, and you might be curious about what the feed looks like for someone else. 

We are proposing adding a feature where you can choose any profile to view the feed through.

The endpoint already exists for this, just need UI/FE development.

----Design to follow today----

### PR DESCRIPTION
**Pull request by sasicodes**
## What does this PR do?

Fixes #1053 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Enhancement (non-breaking small changes to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Checklist

- I haven't read the [contributing guide](https://github.com/lensterxyz/lenster/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by bigint** [comment]
Still I don't see a spinner 😕 

Can we make a different query to fetch only users from the timeline after clicking on the select component?

**Comment by bigint** [comment]
Always make sure to have conventional PR names 🙏🏼 😊

File: apps/web/src/components/Home/SeeThroughLens.tsx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    }
+    setRecommendedProfilesToSeeThrough(profiles?.slice(0, 5));
+  };
+
+  const profile = seeThroughProfile ?? currentProfile;
+  const request = { profileId: profile?.id, limit: 50 };
+
+  const [searchUsers, { data: searchUsersData, loading: searchUsersLoading }] = useSearchProfilesLazyQuery();
+
+  const [fetchRecommendedProfiles, { loading, error }] = useTimelineLazyQuery({
```
Comment (bigint) [inline apps/web/src/components/Home/SeeThroughLens.tsx:53]: Can we make a new query to get only profiles?

Comment (sasicodes) [reply apps/web/src/components/Home/SeeThroughLens.tsx:53]: to get timeline users?

Comment (bigint) [reply apps/web/src/components/Home/SeeThroughLens.tsx:53]: yes

**Review by bigint — APPROVED** [review]

**Pull request merged**